pkg/client/unversioned/client.go

```diff
@@ -198,14 +199,14 @@ func (c *Client) SwaggerSchema(groupVersion string) (*swagger.ApiDeclaration, er
 	}
 	groupVersions := ExtractGroupVersions(groupList)
 	// This check also takes care the case that kubectl is newer than the running endpoint
-	if stringDoesntExistIn(groupVersion, groupVersions) {
-		return nil, fmt.Errorf("API version: %s is not supported by the server. Use one of: %v", groupVersion, groupVersions)
+	if stringDoesntExistIn(version.String(), groupVersions) {
+		return nil, fmt.Errorf("API version: %v is not supported by the server. Use one of: %v", version, groupVersions)
 	}
 	var path string
-	if groupVersion == "v1" {
-		path = "/swaggerapi/api/" + groupVersion
+	if version == v1.SchemeGroupVersion {
+		path = "/swaggerapi/api/" + version.Version
 	} else {
-		path = "/swaggerapi/apis/" + groupVersion
+		path = "/swaggerapi/apis/" + version.Group + "/" + version.Version
```

Comment:
> version.String()?

I'm trying to avoid that to make the final sweep for "are we done" easier.  The fact that `String` gives back something you can plug into a URL isn't something I really want to support.